3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,87 @@
+//Age-ify (A future age calculator)
+let yearOfBirth = 1999;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log('you will be' + " " + age + ' yeras old in' + " " + yearFuture);
+
+// A Dog Age Calculator 
+let dogYearOfBirth = 2017
+let dogYearFuture = 2027;
+let dogYear = "dog";
+
+let shouldShowResultInDogYears = true;
+let dogAge = dogYearFuture - dogYearOfBirth;
+let humanAge = dogAge * 7;
+
+if (shouldShowResultInDogYears) {
+    console.log("your dog will be " + dogAge + " " + dogYear + " " + "years old in" + " " + dogYearFuture)
+} else {
+    console.log("your dog will be " + humanAge + " " + dogYear + " " + "human years old in" + " " + dogYearFuture)
+}
+
```

Comment:
The logic behind calculating dog years is clear.
🚩 A couple of points here:
- The dogYear variable is assigned the string "dog". It could be renamed to dogYearUnit for clarity.
- You can use template literals for cleaner concatenation.
- `const` can be used for variables that don't change.